include/deal.II/lac/constraint_matrix.h

```diff
@@ -1406,6 +1421,8 @@ ConstraintMatrix::add_entry (const size_type line,
   // exists, since we don't want to enter it twice
   Assert (lines_cache[calculate_line_index(line)] != numbers::invalid_size_type,
           ExcInternalError());
+  Assert (!local_lines.size() || local_lines.is_element(column),
```

Comment:
I am only worried introducing this (especially right before the 8.3 release) without knowing if this breaks anything. Maybe I am just too cautious here. :-) 
@kronbichler what do you think, is this change safe?